src/gentropy/sumstat_qc_step.py

```diff
@@ -0,0 +1,36 @@
+"""Step to run a finemapping using."""
```

Comment:
Needs to be changed:

```suggestion
"""Step to calculate quality control metrics on the provided GWAS study."""
```